### DISCUSSION TITLE
What is the bandwidth usage pattern for an audio download?

### DISCUSSION THREAD

**Discussion by adithyaamara** [Tyrrrz/YoutubeDownloader#368] [Q&A] [answered]
All the youtube downloaders I have used till now have been downloading video first, then converting it to audio. [When chosen for MP3 download]. This is obviously wastage of resources.

So, If I select an MP3 download, will this utility download just MP3 (saves my bandwidth). Or does it download a video and post process it to audio?

I am really curious, Please let me know.

@Tyrrrz Anyways, This is single best youtube downloader I have found online. Great work!

**Comment by Tyrrrz** [comment, marked as answer]
If you select `mp3` as the output, YTD will pull the highest quality audio-only stream (`mp4` or `webm`) and covert that to `mp3`. It won't pull any video streams.
